transcom/mymove
# Expose EDI Error details for payment requests in the logs

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16945)

## Summary

This PR adds EDI Error details for payment requests to AWS cloudwatch logs so someone can trace it

## Verification Steps for the Author & Reviewers
Check the AWS cloudwatch logs in order to see if EDI Error logs are being printed.
